src/bip39.en.rs

```diff
@@ -0,0 +1,2055 @@
+// Copyright 2020 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+// https://github.com/trezor/python-mnemonic/tree/master/mnemonic/wordlist
```

Comment:
```suggestion
// https://github.com/bitcoin/bips/blob/master/bip-0039/english.txt
```
Referencing the word lists under bitcoin/bips is maybe a bit cleaner than using the 3rd party Trezor link

Reply:
Nice, I think I was reading an old version of the wiki where they referred to the trezor lists.